### DISCUSSION TITLE
License metadata

### DISCUSSION THREAD

**Discussion by maximlt** [bokeh/bokeh#12661] [General]
For your information as I don't think that's a bug, Bokeh 3 changed the way it's packaging its license data and that made the not-so-pretty [pyviz.org tools dashboard](https://pyviz.org/tools.html) even less pretty :)

![image](https://user-images.githubusercontent.com/35924738/205897289-8e6b8283-030d-4c23-a6cc-9f101aa6d7c8.png)

I think shields.io, the service we use to build these badges, should fix that on their side. I've opened an issue there: https://github.com/badges/shields/issues/8689

(another package - voila - shows the same symptoms in the dashboard)

**Comment by bryevdv** [comment]
Thanks for the upsteam issue @maximlt FYI PyPI has a similar issue: https://pypi.org/project/bokeh/
But i have not had a chance to figure out where to submit an issue yet (maybe just a Packaging post on DPO)

Edit: I missed you noted the PyPI presentation in the other issue and perhaps that is actually the root cause. I'll open a discussion on DPO to ask what is intended here.

**Reply by bryevdv** [reply]
https://discuss.python.org/t/file-values-for-license-metadata/21715

**Reply by bryevdv** [reply]
PyPI is aware and is apparently waiting for PEP 639 to do anything  https://github.com/mesonbuild/meson-python/issues/129

**Comment by bryevdv** [comment]
That said, I guess we could also just set the value to `text = BSD-3-Clause` (I'm assuming that's what it is expecting)  in `pyproject.toml` on our end. cc @bokeh/dev

**Comment by bryevdv** [comment]
Just commenting that this has actually spun out into several discussions across DPO and a few other trackers. However,  in the immediate term the best WAR for us is probably to delete the `license` key and then things will just use the trove classifier for now.
